pmm-app/src/pmm-qan/panel/components/Details/Table/TableContainer.hooks.ts

```diff
@@ -11,14 +11,16 @@ export const useTables = (example, explains, databaseType): any[] => {
     const getTables = async () => {
       setLoading(true);
       setTables([]);
-      if (databaseType === Databases.mysql && jsonExplain.value) {
-        const parsedJSON = JSON.parse(jsonExplain.value);
+      if (databaseType === Databases.mysql && example && jsonExplain) {
+        // eslint-disable-next-line global-require
+        const parser = require('sqlite-parser');
```

Comment:
could be this imported like other libs (like `import { get } from 'lodash';`) to avoid importing this library each time when this code is called?

Reply:
I have another question, is it a workaround of somesort?